spec/models/user_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+describe User do
+  let(:user){ build(:user) }
+
+  # Happy
+  it "is valid with default attributes" do
+    expect(user).to be_valid
+  end
+
+  it "has password length bigger then 4" do
```

Comment:
I would test edge cases on these, meaning make sure 4 and 24 pass. That way you know it is actually testing if 4/24 are valid and just beyond (3/25) is not.